common-content/en/module/js3/identifying-state/index.md

```diff
@@ -33,11 +33,13 @@ In our film example, we would not store "is the search term empty" and "what is
 
 #### 🖇️ If two things always change together, they should be one piece of state.
 
-If our website had light mode and dark mode, we would not have one state for "is dark mode enabled" and one state for "is light mode enabled". We would have one piece of state: a {{<tooltip title="boolean">}}true or false. On or off.{{</tooltip>}}
+If our website had light mode and dark mode, we would not have one state for "is dark mode enabled" and one state for "is light mode enabled". We would have one piece of state: a {{<tooltip title="boolean">}}true or false. On or off.{{</tooltip>}}. We would also not have a state for "background colour" - we would derive that from the one boolean.
```

Comment:
```suggestion
If our website had light mode and dark mode, we would not have one state for "is dark mode enabled" and one state for "is light mode enabled". We would have one piece of state: a {{<tooltip title="boolean">}}true or false. On or off.{{</tooltip>}}. 

We would also not have a state for "background colour". We would derive that from the single boolean: light: TRUE/FALSE.
```

I'm pondering this. I mean it's true - that's how you write dark modes HOWEVER in a browser you always have two competing states -- there's the browser state: prefers dark mode, and then there's the override button, which you want users to have because there's always a significant minority who, for example, prefer darkmode generally but not YOUR darkmode, or vv. And there isn't a default position to start.

Therefore if they Inspect the curriculum, or any reasonably complex site, they will likely find two state settings handling dark/light, both `@media(prefers-color-scheme: dark) ` or light, to set up the initial colours, and then a class is-light-mode / is-dark-mode to add specificity to override the default preference on user selection.

And of course if they Inspect poorly written sites, of which there are many, they will see things like `.is-dark-mode {background: var(--dark-background-color);}` which is obviously heinous and mad but common. 

So is this example clarifying or potentially confusing? (I'm aware you're elaborating on an existing example --and it's the existing example I am questioning here, piqued by your elaboration)

Reply:
I think this is getting pretty in the weeds. But I take the point. I replaced this example with "is a user logged in" + "who is currently logged in" - does that work for you?